src/version.rs

```diff
@@ -300,16 +300,17 @@ pub fn dependencies(req: &mut Request) -> CargoResult<Response> {
 
 /// Handles the `GET /crates/:crate_id/:version/downloads` route.
 pub fn downloads(req: &mut Request) -> CargoResult<Response> {
+    let (offset, limit) = req.pagination(90, 100)?;
```

Comment:
I'm not sure that the concept of a `page` makes as much sense here. Perhaps we should just have the parameter be called `before_date`, which takes the date directly, and only allow the 90 day interval.